migrations/2023-09-01-072744_payment_methods_add_card_details/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS card_details BYTEA DEFAULT NULL;
```

Comment:
Since we are renaming the column in the subsequent migration, this migration is not required. We can just have one migration that creates the `payment_method_data` column.